scripts/project-catalogue/services/ontology.py

```diff
@@ -0,0 +1,97 @@
+import requests
+from urllib.parse import quote
+
+
+KNOWN_DISCREPANCIES = {
+    "10X v3 3'": 'EFO_0009922',
+    "10X v2 3'": 'EFO_0009899'
+}
+
+
+class QuickOntology:
+    def __init__(self, url: str = None):
+        if not url:
+            url = 'https://www.ebi.ac.uk/ols/api'
+        self.url = url.rstrip('/')
+        self.organ_not_found = []
+        self.tech_not_found = []
+        self.nearest = {}
+
+    def get_organ(self, search: str):
+        params = {
+            'groupField': 'iri',
+            'allChildrenOf': 'http://purl.obolibrary.org/obo/UBERON_0000465',
+            'ontology': ['hcao', 'uberon']
+        }
+        organ = self.ontology_search(search, params)
+        if organ:
+            return organ
+        if search not in self.organ_not_found:
+            self.organ_not_found.append(search)
+            print(f'No ontology found for organ: {search}')
```

Comment:
Is this a result? A user informative message? Have you considered using `logging` package?

Reply:
Work in progress code, now resolved